js/ui/lookup.js

```diff
@@ -233,6 +233,14 @@ const Lookup = DropDownList.inherit({
             * @hidden
             */
 
+            /**
+             * @name dxLookupOptions.disableOptionCentering
+             * @type boolean
+             * @default false @for Material
+             * @public
+             */
+            disableOptionCentering: true,
```

Comment:
Has it been discussed with @RomanTsukanov  or @AlexJameson ?
Almost all of our component options are permissive, not prohibitive (`allowSomething`, `featureEnabled` etc)

Reply:
You've got a point. Maybe `itemCenteringEnabled`?

Reply:
Done